latex/chapter/Konzeption.tex

```diff
@@ -167,3 +167,54 @@ \subsubsection{Software Architektur}
 
 % Beschreibung der Architektur
 % GGF Sequenzdiagramm
+
+\subsection{Demosystem}
+Das Demosystem baut auf dem Versuchssystem auf.
+Wie in den Anforderungen beschrieben, ist die Authentifizierung und damit auch das Versuchssystem Teil des Back-Ends.
```

Comment:
Evtl. Ergänzung: klassische Webapllikation in der Client Server Architektur

Reply:
Ja, hab ich mir auch noch gedacht, ich überarbeite jetzt gleich noch ein bisschen und ergänze noch was.